### DISCUSSION TITLE
Add GeoPackages

### DISCUSSION THREAD

**Discussion by jvanulde** [OpenDRR/earthquake-scenarios#13] [Ideas]
@tieganh, @DamonU2, @wkhchow we should consider adding the final GeoPackage (GPKG) datasets for each scenario in this repository. My view is that those who fork the repository are likely expecting the datasets to come along with it. Thoughts?

**Comment by DamonU2** [comment]
Generate geopackage files for scenarios in finished directory - https://github.com/OpenDRR/earthquake-scenarios/issues/14
